<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketDialog</class>
 <widget class="QDialog" name="TicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>366</height>
   </rect>
  </property>
  <widget class="QTableView" name="m_ticketList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>291</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="m_createTicket">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Продатьбилет на рейс</string>
   </property>
   <property name="text">
    <string>Оформить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_moveButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Переоформить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_registerButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегистрировать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
